config-linux.md

```diff
@@ -490,9 +490,9 @@ For more information, see [the man page](http://man7.org/linux/man-pages/man8/sy
    }
 ```
 
-## seccomp
+## Seccomp
 
-Seccomp provides application sandboxing mechanism in the Linux kernel.
+**`seccomp`** (object, OPTIONAL) provides application sandboxing mechanism in the Linux kernel.
```

Comment:
I don't feel this is correct, mainly because "Seccomp" here is referring to the kernel feature not the object in the spec. One of the other "Seccomp" references should be changed.